src/io.jl

```diff
@@ -51,5 +51,11 @@ function read(fn::Union{AbstractString,Parquet2.FilePathsBase.AbstractPath,Parqu
     for column in keys(meta.columns)
         df[!, column] = GFT.WellKnownBinary.(Ref(GFT.Geom()), df[!, column])
     end
+    # set GeoInterface metadata
+    metadata!(df, "GEOINTERFACE:geometrycolumns", Tuple(Symbol.(keys(meta.columns))))
+    crs = meta.columns[meta.primary_column].crs
```

Comment:
Yes is this just a string? May need to wrap it with GFT like WellKnownText

Reply:
No, in this line, `crs` is a `Dict`